app/Http/Controllers/API/UserController.php

```diff
@@ -17,15 +17,38 @@
 namespace App\Http\Controllers\API;
 
 use App\Http\Resources\UserResource;
+use App\Models\BonTransactions;
+use Illuminate\Support\Facades\DB;
 
 class UserController extends BaseController
 {
     final public function show(): UserResource
     {
         $user = auth()->user();
 
+        $history = DB::table('history')
+            ->where('user_id', '=', $user->id)
+            ->where('created_at', '>', $user->created_at)
+            ->selectRaw('SUM(actual_uploaded) as upload_sum')
+            ->selectRaw('SUM(uploaded) as credited_upload_sum')
+            ->selectRaw('SUM(actual_downloaded) as download_sum')
+            ->selectRaw('SUM(downloaded) as credited_download_sum')
+            ->selectRaw('SUM(seedtime) as seedtime_sum')
+            ->selectRaw('SUM(actual_downloaded > 0) as download_count')
+            ->selectRaw('COUNT(*) as count')
+            ->first();
+
+        $seedingSize = $user->seedingTorrents()->sum('size');
+
+        $bonusUploaded = BonTransactions::query()
+            ->where('sender_id', '=', $user->id)
+            ->where('name', 'like', '%Upload%')
+            ->sum('cost');
```

Comment:
```suggestion
        $bonusUploaded = BonTransactions::query()
            ->where('sender_id', '=', $user->id)
            ->where('name', 'like', '%Upload%')
            ->sum('cost');
```

Wildcard searches shouldn't be permitted in the api.